### PR TITLE
Rename the node with pipeline

### PR DESCRIPTION
**Pull request by shiyu22**

### CODE REVIEW

File: towhee/runtime/operator_manager/operator_action.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from towhee.runtime.dag_repr import DAGRepr
```
Comment (junjiejiangjjj) [inline towhee/runtime/operator_manager/operator_action.py:15]: `DAGRepr`   <--> `OperatorAction`  we'd better remove the circular reference.

Comment (junjiejiangjjj) [inline towhee/runtime/operator_manager/operator_action.py:15]: Can you remove the dependency of DAGRepr.

Comment (shiyu22) [reply towhee/runtime/operator_manager/operator_action.py]: Done.

**Pull request merged**